src/routes/onboarding/set-medication-reminders/medication-reminder-form.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Form } from "../../../components"
+import { MedicationSection } from "./medication-section";
+
+export const MedicationReminderForm: React.FC = () => {
+  return (
+    <div className="flex flex-col bg-lightYellow max-w-screen-md pb-8 px-[10px] relative m-auto">
+      <h2 className="font-text text-section-header font-medium leader-[38.4px] text-center max-w-[358px] mx-auto">Set your medication reminders</h2>
+      <p className="font-text text-center max-w-[358px] mx-auto mt-4 mb-8">It’s important to be consistent in your medications, and notifications may help!</p>
+      <Form className="flex flex-col gap-5 ">
+        <Form.AddMoreSection buttonText="+ Add another medication">
```

Comment:
There is too much spacing between this button and the bottom medication section. The Figma doc says that there should be 20px between the two elements.

Your implementation
![image](https://github.com/user-attachments/assets/b73e3c8b-16c1-4a0f-8a45-7332e9c98806)

Figma Docs
![image](https://github.com/user-attachments/assets/5f9a33dc-c134-4e3d-bbbc-42cb4342205b)
![image](https://github.com/user-attachments/assets/407e7b4c-d39b-48fb-aaf1-9502f8078a8f)